app/assets/javascripts/components/company_overview.es6.jsx

```diff
@@ -0,0 +1,73 @@
+class CompanyOverview extends React.Component {
+  render () {
+    return (
+        <div className="card" id="overview">
+            <div className="card-header">
+                <h4 className="card-title">
+                    Overview
+                </h4>
+            </div>
+            <div className="card-content">
+                <div className="card-body">
+                    <div className="row">
+                        <div className="card-text col-sm-2">
+                            <img src={`${this.props.logo}`} alt={`${this.props.company.name}`} className="company-logo"/>
+                        </div>
+                        <div className="card-text col-sm-10">
+                            <h4>{this.props.company.name}</h4>
+                            <a href={`${this.props.url_domain}`} target="_blank">{this.props.url_domain}</a>
+                        </div>
+                    </div>
+                    <div className="row">
+                        <div className="card-text col-sm-2">
+                            <h6>Description</h6>
+                        </div>
+                        <div className="card-text col-sm-10">
+                            <p>{this.props.company.overview}</p>
+                        </div>
+                    </div>
+                    <div className="row">
+                        <div className="card-text col-sm-2">
+                            <h6>Founded Year</h6>
+                        </div>
+                        <div className="card-text col-sm-10">
+                            <p>{this.props.company.founded}</p>
+                        </div>
+                    </div>
+                    <div className="row">
+                        <div className="card-text col-sm-2">
+                            <h6>Count of Employees</h6>
+                        </div>
+                        <div className="card-text col-sm-10">
+                            <p>{this.props.company.approx_employees}</p>
+                        </div>
+                    </div>
+                    <div className="row">
+                        <div className="card-text col-sm-2">
+                            <h6>Industries</h6>
+                        </div>
+                        <div className="card-text col-sm-10">
+                            <ul>
+                                {this.props.industries.sort(function(a, b){return (a.name > b.name) ? 1 : ((b.name > a.name) ? -1 : 0);}).map(function(x){
```

Comment:
it is really horrible to read)

Reply:
agree, i was thinking of using more javascript instead of passing industries sorted with ruby. but that wasn't a good choice

Reply:
js is terrible in itself))
so it is better to use lodash
https://lodash.com/docs/4.17.5#sortBy